x-pack/plugins/enterprise_search/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "enterpriseSearch",
+  "version": "1.0.0",
```

Comment:
Quick check, is this the right version? Just looking at some other plugins and they are all set to 8.0.0.

Reply:
I believe the correct version is:
```
"version": "M.V.P",
```
:trollface:

Reply:
Ah good point, the only other plugin(s) I can see that are different are ones that specify `"version": "kibana",` and `event_log` for some reason. Will check in the Slack channel

Reply:
Per the inestimable Josh Dover, this should be `"version": "kibana"` 👍

Reply:
HAHA oh man Scotty's comment only just showed up on hard refresh. Would delayed laugh again, A+++